### PR TITLE
[MIG][9.0] base_export_manager

### PR DESCRIPTION
**Pull request by lasley**
This PR migrates base_export_manager to v9

No changes were required outside of the standard upgrades. I did notice a remnants of the old naming in the JS, which I resolved while v9-ifying it

### CODE REVIEW

**Comment by rafaelbn** [comment]
@lasley thanks! but we mus wait for this module... it's still pending an improvement in v8 very important https://github.com/OCA/server-tools/pull/317

**Comment by lasley** [comment]
Thanks for the pointer towards the open 8.0 PR @rafaelbn - seems I missed that. I will review that PR sometime today & bring this way when merged.

**Comment by rafaelbn** [comment]
Hi @lasley , https://github.com/OCA/server-tools/pull/317 is ready to be merged after that we can continue with this one.

**Comment by rafaelbn** [comment]
Hi @lasley , https://github.com/OCA/server-tools/pull/317 is merged. Could you bring please all to this PR in order to test it? Thanks!

**Comment by lasley** [comment]
Rebased

File: base_export_manager/models/ir_exports.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    name = fields.Char(required=True)
+    resource = fields.Char(
+        required=False,
+        readonly=True,
+        help="Model's technical name.")
+    model_id = fields.Many2one(
+        "ir.model",
+        "Model",
+        store=True,
+        domain=[("osv_memory", "=", False)],
```
Comment (pedrobaeza) [inline base_export_manager/models/ir_exports.py]: This domain is no longer valid in v9

**Comment by lasley** [comment]
Thanks for the catch @pedrobaeza - fixed!

**Comment by rafaelbn** [comment]
Super! Tested in runbot! :+1:

**Comment by max3903** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**